vaadin-lumo-theme-flow-parent/vaadin-lumo-theme-flow/src/main/java/com/vaadin/flow/theme/lumo/LumoUtility.java

```diff
@@ -18,6 +18,8 @@
 /**
  * Contains the definition for all the CSS utility classes provided by Lumo.
  */
+@CssImport("./lumo-utility-classes.css")
+@NpmPackage(value = "lumo-css-framework", version = "^4.0.10")
```

Comment:
If I remember correctly, this npm package isn't the current lumo utility classes. It was more like an experiment / proof of concept / idea by @anezthes before it was officially added inside the web component's repo

Reply:
Thanks. The original `lumo-css-framework` npm package should be probably deprecated to make it more clear.